### PR TITLE
Fix for input and textarea bug

### PR DESCRIPTION
**Pull request by PavlovicWorkCo**
This PR should fix Issue #920 . 
A bit more detail about the issue:
Inputs and text areas are not handled properly.
In cases in which we want to have multiple text inputs in one custom plugin, some of them with formatable text and some of them without, we need to use inputs and/or text areas (for those which we don't want to have formatable text). In the current version this causes multiple bugs when hitting enter in inputs and textareas, because the `currentInput` is not updated properly and because the text in inputs and textareas is not innerHTML (it's in their value attribute).
Bugs:
1) It copy pastes the whole `<input>` element and puts it in the next paragraph outside of the block instead of only cutting out the text from the caret to the end. 
2) It cuts out the `<input>` element (without the text in it) and copies the whole content editable element.
input-bugs

### CODE REVIEW

**Comment by hata6502** [comment]
I support this PR!

**Comment by PavlovicWorkCo** [comment]
@neSpecc @khaydarov @gohabereg Hi. Could you take a look at the issue and PR, and let me know if anything needs changing and if and when you would consider merging it. This bug is a blocker for a project that is soon going into production, so we would very much appreciate if it could be resolved without having to fork the repo. 🙏

**Comment by neSpecc** [comment]
> @neSpecc @khaydarov @gohabereg Hi. Could you take a look at the issue and PR, and let me know if anything needs changing and if and when you would consider merging it. This bug is a blocker for a project that is soon going into production, so we would very much appreciate if it could be resolved without having to fork the repo. 🙏

we will check it soon. The next release is planned for the end of July.

File: src/components/utils.ts [hunk 1]
```diff
@@ -556,3 +556,12 @@ export function getValidUrl(url: string): string {
 export function openTab(url: string): void {
   window.open(url, '_blank');
 }
+
+/**
+ * Checks if element is "<input />" or "<textarea />"
+ *
+ * @param element - Element to check
+ */
+export function isNativeTextInput(element: Node | HTMLElement): boolean {
```
Comment (neSpecc) [inline src/components/utils.ts:565]: we already have such util, see `dom.ts@isNativeInput`

**Review by neSpecc — CHANGES_REQUESTED** [review]

**Review by gohabereg — APPROVED** [review]

**Review by neSpecc — APPROVED** [review]

**Pull request merged**